molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -47,17 +58,60 @@
 class RestControllerV2
 {
 	private static final Logger LOG = LoggerFactory.getLogger(RestControllerV2.class);
+	static final int MAX_ENTITIES = 1000;
```

Comment:
How long does the Location url get for the standard ID generator and 1000 entities? Do we expect that to fit in all browsers?